include/sentry.h

```diff
@@ -1813,6 +1813,29 @@ SENTRY_API sentry_attachment_t *sentry_scope_attach_file(
 SENTRY_API sentry_attachment_t *sentry_scope_attach_file_n(
     sentry_scope_t *scope, const char *path, size_t path_len);
 
+/**
+ * Attaches bytes to be sent along with events.
+ *
+ * `filename` is assumed to be in platform-specific filesystem path encoding.
+ * API Users on windows are encouraged to use `sentry_attach_bytesw` or
+ * `sentry_scope_attach_bytesw` instead.
```

Comment:
there could be a note about that crashpad requires unique filenames, so the attachment may end up renamed to `basename-N.ext` if necessary.

Reply:
Yes, this makes sense.